.circleci/config.yml

```diff
@@ -458,13 +460,6 @@ jobs:
           key: yarn-docs-{{ checksum "website/yarn.lock" }}
           paths:
             - ~/.cache/yarn
-      - run:
-          name: Pack docs
-          command: tar -czvf protostar-docs.tar.gz website/build
-      - persist_to_workspace:
-          root: .
-          paths:
-            - protostar-docs.tar.gz
```

Comment:
Why was this step removed?

Reply:
they are not needed at the moment, as we still deploy docs from github (and since docs will be built for every PR, it is just a waste of credits/storage to keep all the built versions of docs for no real reason). There is a task to deploy docs from circle, it will be re-added then.